app/src/main/java/com/sopt/now/compose/ui/screens/home/HomeViewModel.kt

```diff
@@ -1,95 +1,109 @@
-        ),
-        Friend(
-            profileImage = R.drawable.ic_launcher_foreground,
-            name = "우상욱",
-            selfDescription = "나보다 안드 잘하는 사람 있으면 나와봐",
-        ),
-        Friend(
-            profileImage = R.drawable.ic_launcher_foreground,
-            name = "배지현",
-            selfDescription = "표정 풀자 ^^",
-        ),
-        Friend(
-            profileImage = R.drawable.ic_launcher_foreground,
-            name = "최준서",
-            selfDescription = "애들아 힘내보자고~!!",
-        ),
-        Friend(
-            profileImage = R.drawable.ic_launcher_foreground,
-            name = "이유빈",
-            selfDescription = "나보다 귀여운 사람 나와봐",
-        ),
-        Friend(
-            profileImage = R.drawable.ic_launcher_foreground,
-            name = "김언지",
-            selfDescription = "오비의 맛을 보여줘야겠고만!",
-        ),
-        Friend(
-            profileImage = R.drawable.ic_launcher_foreground,
-            name = "박동민",
-            selfDescription = "컴포즈는 나에게 맡기라고!",
-        ),
-        Friend(
-            profileImage = R.drawable.ic_launcher_foreground,
-            name = "배찬우",
-            selfDescription = "마지막 20대를 불태워 보겠어..",
-        ),
-        Friend(
-            profileImage = R.drawable.ic_launcher_foreground,
-            name = "송혜음",
-            selfDescription = "소주잔 3조 화이팅!!",
-        ),
-        Friend(
-            profileImage = R.drawable.ic_launcher_foreground,
-            name = "박유진",
-            selfDescription = "코리조는 3조가 최고~",
-        ),
-        Friend(
-            profileImage = R.drawable.ic_launcher_foreground,
-            name = "이석준",
-            selfDescription = "내가 제일 잘나가ㅏ",
-        )
-    )
+class HomeViewModel(
+    private val followerRepository: FollowerRepository,
+    private val authRepository: NetworkMemberRepository
+) : ViewModel() {
+
+    private val _uiState = MutableStateFlow<HomeUiState>(HomeUiState.Loading())
+    val uiState: StateFlow<HomeUiState> = _uiState.asStateFlow()
 
-    fun updateUiState(user: User?) {
-        if(user != null) {
-            _uiState.value = HomeUiState.Success(user)
+    private fun updateUiState(
+        state: HomeUiState.Loading = _uiState.value as HomeUiState.Loading
+    ) {
+        if (state.isUserSuccess && state.isFollowerSuccess) {
+            _uiState.value = HomeUiState.Success(
+                user = state.user,
+                follower = state.follower
+            )
         } else {
             _uiState.value = HomeUiState.Error
         }
     }
 
-    fun fetchUserLoggedIn(navController: NavHostController) {
-            navController.previousBackStackEntry?.savedStateHandle
-                ?.getLiveData<User>(NAVIGATE_LOGIN_KEY)?.value.run { updateUiState(this) }
+    fun fetchNetworkData() {
+        fetchUserInfo()
+    }
 
+    private fun fetchUserInfo() {
+        viewModelScope.launch {
+            val result = authRepository.getUserInfo()
+            result.fold(
```

Comment:
```suggestion
            authRepository.getUserInfo().fold(
```

변수 인라인할 수 있으면 인라인으로 적어주세요